src/screens/Home/index.tsx

```diff
@@ -2,34 +2,47 @@ import React from 'react';
 import {FlatList, StatusBar} from 'react-native';
 import {
   Container,
-  FlatListContainer,
   BackgroundImage,
   HeaderTitle,
+  FlatListContainer,
 } from './styles';
 import PokemonCard from './localComponents/PokemonCard';
 import {apiJSON} from '~/utils/apiMockData';
 import Separator from '~/components/Separator';
-import HeaderLabel from './localComponents/HeaderLabel';
+import Header from '~/components/Header';
 import pokeBallImage from '~/assets/images/pokeball.png';
+import {useNavigation} from '@react-navigation/native';
+import ButtonHeader from '~/components/ButtonHeader';
 
 const Home: React.FC = () => {
+  const {navigate} = useNavigation();
+  const handlerNavigateInternal = () => {
```

Comment:
```suggestion
  const handleNavigateToInternal = () => {
```

Change to something like this

Reply:
resolved in commit 976fa0aa6c578fbf09030956cfa3018741ea05c8